website/docs/r/nsxt_network_imported.html.markdown

```diff
@@ -0,0 +1,112 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_network_imported"
+sidebar_current: "docs-vcd-resource-nsxt-network-imported"
+description: |-
+  Provides a VMware Cloud Director Org VDC imported Network. This can be used to create, modify, and
+  delete imported VDC networks (backed by NSX-T).
+---
+
+# vcd\_nsxt\_network\_imported
+
+Provides a VMware Cloud Director Org VDC imported Network. This can be used to create, modify, and
```

Comment:
I have an ask. I'm looking at this page and would like to ask for a clearer distinction between NSX-T imported type vs. Terraform imported resource. I think it's very easy for new users to get confused when they see "imported" all over the place.

I think in this paragraph we should explain that it is "NSX-T imported type network" and what exactly this means (that it is backed by a segment).

Also, where we say "imported network", maybe we could say "NSX-T imported type network" to be explicit?

Reply:
Noted. I have made an attempt to improve this and datasource docs. Tried to differentiate these concepts and popped an additional note with link to official VCD docs. Looks like that in preview now:
<img width="1054" alt="image" src="https://user-images.githubusercontent.com/15804230/109429596-2179fd00-7a05-11eb-8dd3-62a6374de3f8.png">

Reply:
It's great! Only one ask: please make the added notes blue instead of yellow - blue is enough in this case.

Reply:
Changed in resource and data source
<img width="752" alt="image" src="https://user-images.githubusercontent.com/15804230/109499308-618cbe80-7a9d-11eb-9916-d21f725e99cb.png">